lib/quickbooks-ruby.rb

```diff
@@ -219,4 +228,12 @@ def initialize(msg)
     end
   end
 
+  class Configuration
+    def self.set_tokens_and_realm_id token, secret, realm_id
+      Thread.current[:token]    = token
```

Comment:
In addition to @ruckus comment: Thread.current is useful when you expect that multiple threads can have different tokens. So we cannot use a global variable in that case.